docs/_posts/2016-01-11-installation-guide.md

```diff
@@ -1,12 +1,13 @@
 ---
 layout: page
-title: "Administrator's Manual"
+title: "Installation Guide"
```

Comment:
Is there a different term that would make it clear that this also contains instructions for upgrading MISO? (That's the `Releasing and Upgrading` section near the bottom. Maybe moving `Upgrading MISO to a new version` to its own page?)

Reply:
I can also put it in the new Admin manual.

Reply:
After reading it again, I think you're right and it should go into its own post.

Reply:
Re-reading the docs, would it be better as "Build Guide" / "Deployment guide" / "Admin manual"?